### DISCUSSION TITLE
All tasks are stopped in the scheduled state.

### DISCUSSION THREAD

**Discussion by cjungm** [apache/airflow#20307] [Q&A]
Apache Airflow version:
1.10.12

Environment:
Amazon Linux2
Redis (AWS Elasticache), CeleryExecutor, EFS

What happened:
All processes are running.
Web server, scheduler, and worker processes are all normal.
All Dags and Tasks are successfully scheduled and triggered. But tasks status are never being running.
When I restart scheduler process, tasks become running status.
After tasks become pending status, no task become running.
All tasks are pending

What you expected to happen:
All tasks should be running normally

scheduler health status
`{
"metadatabase": {
"status": "healthy"
},
"scheduler": {
"status": "healthy",
"latest_scheduler_heartbeat": "2021-12-15T01:53:08.770845+00:00"
}
}`

**Comment by potiuk** [comment]
While i cannot help you you should be aware that Airflow 1.10 is end of life as of June 2021. It won't receive even critical security fixes. While Airflow is not vulnerable to the recent log4Shell vulnerability, you make yourself and your employee vulnerable to any other issues that could be found.

Also everyone is now focused on Airflow 2 so you are unlikely to get any help with 1.10.12.

Please upgrade to Airflow 2 ASAP.
